base/vulkanexamplebase.cpp

```diff
@@ -18,7 +21,7 @@ VkResult VulkanExampleBase::createInstance(bool enableValidation)
 	appInfo.pEngineName = name.c_str();
 	// Temporary workaround for drivers not supporting SDK 1.0.3 upon launch
 	// todo : Use VK_API_VERSION
-	appInfo.apiVersion = VK_MAKE_VERSION(1, 0, 2);
+	appInfo.apiVersion = VK_MAKE_VERSION(1, 0, 5);
```

Comment:
Why did you do this? I could be mistaken, but I don't think this can affect the code, but it will affect at runtime the ability for users to link the app against older (but compatible) drivers.